doc/README.md

```diff
@@ -1,52 +1,53 @@
-# Tera文档专区
+# Tera Docs
 
-## 简介
+## Introduction
 
-[系统设计](tera_design.md)
+[tera design](tera_design.md)
 
-[Tera表格数据模型及实现](data_model.md)
+[data model and implementation](data_model.md)
 
-[master、ts、client的交互](master-ts-client-interactive.md)
+## User Manual
 
-## 使用帮助
+[pseudo distributed mode](onebox.md)
 
-[体验单机Tera](onebox.md)
+[distributed mode](cluster_setup.md)
 
-[命令行工具teracli使用方法](teracli.md)
+[API manual](sdk_dev_guide.md)
 
-[主要api使用方法](sdk_dev_guide.md)
+[teracli manual](teracli.md)
 
-[搭建tera集群](cluster_setup.md)
+[flag manual](tera_flag.md)
 
-[各flag配置项含义介绍](tera_flag.md)
+## Performance Profiling
 
-## 性能优化
+[Read/Write](perf_opz.md)
 
-[针对表格不同读写特性的性能优化建议](perf_opz.md)
+## Inside Tera
 
-## 功能实现
+[how load balance works](load-balance.md)

```

Comment:
"how load balance works" sounds like to explain why the load balance works (which is challenged by someone), not to explain how it implemented